apps/frontend/src/screens/Game.tsx

```diff
@@ -67,7 +68,7 @@ export const Game = () => {
   >(null);
   const [player1TimeConsumed, setPlayer1TimeConsumed] = useState(0);
   const [player2TimeConsumed, setPlayer2TimeConsumed] = useState(0);
-
+  const [gameID,setGameID] = useState("");
```

Comment:
why this?

doesnt `const { gameId } = useParams();` work?

Reply:
No ser, that won't work. When the first player initially joins he will be at the route /game/random. After receiving the game_added message we will actually get the game ID, using which we create the link ..

Reply:
Agreed , will convert to a normal var.

Reply:
No ser, that won't work. When the first player initially joins he will be at the route /game/random. After receiving the game_added message we will actually get the game ID, using which we create the link ..

Reply:
oh, i see the login is different, would be better if we push the user who is wainting to the actual url with the ID: /game/id... 

for now mergin this.